packages/styles/src/components/utilities.scss

```diff
@@ -67,9 +67,6 @@ span.spacer {
 @each $name, $color in color.$background-colors {
   .bg-#{$name} {
     --post-bg-rgb: #{color-fn.rgb-values($color)};
-    $shadow-color: sass-color.adjust($color, $lightness: -15%);
```

Comment:
This seems like a breaking change, although it's very likely that no one is using these variables...

Reply:
I discussed it with @gfellerph.
This was mistakenly added in the elevation PR. And needs to be removed again.